server/server.js

```diff
@@ -60,33 +60,29 @@ app.post(`${apiBaseUrl}/employer-form`,
 app.post(`${apiBaseUrl}/employee-form`,
   asyncMiddleware(async (req, res) => {
     await validate(EmployeeFormSchema, req.body); // Validate submitted form against schema
-    const formsCollection = dbClient.db.collection(collections.FORMS);
 
     // Generate unique random hex id
     async function generateRandomHexId() {
       const randomHexId = randomBytes(4).toString('hex').toUpperCase();
 
       // Query database do make sure id does not exist, avoiding collision
-      if (await formsCollection.countDocuments({ id: randomHexId }, { limit: 1 }) > 0) {
+      if (await dbClient.db[collections.FORMS].countDoc({ body: { formId: randomHexId } }) > 0) {
         return generateRandomHexId();
       }
       return randomHexId;
     }
 
     // Form ID
-    const id = await generateRandomHexId();
+    const formId = await generateRandomHexId();
```

Comment:
As we discussed, we may not need this. Removing this logic would also avoid the inherent race condition of a colliding ID being created between querying for the generated ID and writing the item to the DB. We could use something like `id UUID PRIMARY KEY DEFAULT gen_random_uuid()` or even use the default ID generated by Massive instead.

Reply:
Yes, I also think this is not necessary since the id generated by massive.js is also simple to read. (a simple increment integer). Are we fine removing this feature in this branch?

Reply:
Totally good with removing this! Let's streamline